packages/editor/src/components/parts/rest/rest-request/rest-target/RestProperties.tsx

```diff
@@ -10,16 +9,14 @@ export const RestProperties = () => {
   const knownProperties = useMeta('meta/rest/properties', undefined, []).data;
 
   return (
-    <PathCollapsible
-      label='Properties'
-      path='properties'
-      defaultOpen={!deepEqual(config.target.properties, defaultConfig.target.properties)}
-    >
+    <PathContext path='properties'>
```

Comment:
Because the PropertyTable is also used for the Accept-Properties and there was no Path set. There was only one path set called "headers".
![grafik](https://github.com/axonivy/inscription-client/assets/141223521/77eb266c-b7a5-40ad-bc11-155145e012c5)

Reply:
Ah didn't saw that. 👍